app/src/main/java/mod/hey/studios/code/SrcCodeEditor.java

```diff
@@ -360,6 +361,7 @@ private void initializeLogic() {
         setTitle(getIntent().getStringExtra("title"));
 
         editor.setTypefaceText(Typeface.MONOSPACE);
+        editor.setPinLineNumber(true);
```

Comment:
Like I said in #487, [this'd be better as Code Editor setting](https://github.com/Sketchware-Pro/Sketchware-Pro/pull/487#pullrequestreview-1033561995).